migrations/20181009110334_added_changelog_table.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.dropTableIfExists('changelog')
+    .createTable('changelog', table => {
+      table.uuid('id').primary().notNull();
+      table.jsonb('message').notNull();
+      table.timestamps(false, true);
+      table.dateTime('deleted');
```

Comment:
Do we want / need to be able to soft-delete changelog records?

Reply:
no, we don't particularly care, but as it extends the `BaseModel` it will throw an error if deleted column is not set (we monkey-patch query to include a `where({ deleted: null })`

Reply:
ok, works for me.